pkg/clusterlogging/objectbucketclaim.go

```diff
@@ -0,0 +1,298 @@
+package clusterlogging
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/golang/glog"
+	"github.com/openshift-kni/eco-goinfra/pkg/clients"
+	"github.com/openshift-kni/eco-goinfra/pkg/msg"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	goclient "sigs.k8s.io/controller-runtime/pkg/client"
+
+	noobaav1alpha1 "github.com/kube-object-storage/lib-bucket-provisioner/pkg/apis/objectbucket.io/v1alpha1"
+)
+
+// ObjectBucketClaimBuilder provides struct for the objectBucketClaim object.
+type ObjectBucketClaimBuilder struct {
```

Comment:
According to the documentation this is storage object.
https://rook.io/docs/rook/latest-release/Storage-Configuration/Object-Storage-RGW/ceph-object-bucket-claim/
I think this object has to be under storage package.